operator/templates/node-exec-rbac.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: rbac.authorization.k8s.io/v1
 kind: Role
 metadata:
-  name: {{ .Name }}-node-backup-role
+  name: {{ .Name }}-node-exec-role
```

Comment:
can we rename it to an even more generic name? {{ .Name }}-node-role, same for sa
with bootstrap feature, we need the permissions to read/update the configmap in our namespace. Instead of adding another sa I think we can enhance one based on enabled features

Reply:
Yeah, i guess it makes sense. I don't think the bootstrap feature can be disabled, right? Otherwise i'd say we should use a separate rbac set. But if it's always on, we should use the same set.

I've renamed the role/rb/sa